src/lensPath.js

```diff
@@ -0,0 +1,27 @@
+var _curry1 = require('./internal/_curry1');
+var assocPath = require('./assocPath');
+var lens = require('./lens');
+var path = require('./path');
+
+/**
+ * Returns a lens whose focus is the specified path.
+ *
+ * @func
+ * @memberOf R
+ * @category Object
+ * @typedefn Lens s a = Functor f => (a -> f a) -> s -> f s
+ * @sig [String] -> Lens s a
+ * @param {Array} path The path to use.
+ * @return {Lens}
+ * @see R.view, R.set, R.over
+ * @example
+ *
+ *      var xyLens = R.lensPath(['x', 'y']);
+ *
+ *      R.view(xyLens, {x: {y: 2, z: 3}});            //=> 2
+ *      R.set(xyLens, 4, {x: {y: 2, z: 3}});          //=> {x: 4, y: 2}
+ *      R.over(xyLens, R.negate, {x: {y: -2, z: 3}}); //=> {x: -1, y: 2}
+ */
+module.exports = _curry1(function lensPath(p) {
+  return lens(path(p), assocPath(p));
+});
```

Comment:
Just curious whether you considered the alternative implementation:

```js
module.exports = compose(apply(compose), map(lensProp));
```

I'm not saying it's any better.  It's just an interesting possibility, building these by composing `lensProp` instances.